docs/real-device-config.md

```diff
@@ -126,11 +126,10 @@ is updated, and is _not_ recommended):
     `appium/node_modules/appium-xcuitest-driver/node_modules`) and, if it is not present there, under `appium/node_modules`.
     You could also see the full path to WebDriverAgent's folder in Appium server logs
     after a session has been started.
-    Open a terminal and go to that location, then run the following in order to
+    Open a terminal and go to that location, then run the following to
```

Comment:
I would rather add a note about this step is not needed anymore since Appium 1.20, because people could use earlier Appium versions while reading the doc

Reply:
@KazuCocoa and you approved the PR. Should I apply your suggestion nevertheless?

Reply:
yep, adding such comment would still make sense.

Also, I've merged and published the new major appium-webdriveragent component version, so we could bump it in package.json

Reply:
Changed it.

Reply:
documentation is in sync with current package.json which uses 2.32.0. I can create another PR for updating the dependency if you want.

Reply:
thanks